packages/@react-spectrum/accordion/test/Accordion.test.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Accordion, Item} from '../src';
+import {act, fireEvent, render} from '@testing-library/react';
+import {Provider} from '@react-spectrum/provider';
+import React from 'react';
+import {theme} from '@react-spectrum/theme-default';
+import userEvent from '@testing-library/user-event';
+
+let items = [
+  {key: 'one', title: 'one title', children: 'one children'},
+  {key: 'two', title: 'two title', children: 'two children'},
+  {key: 'three', title: 'three title', children: 'three children'}
+];
+
+function renderComponent(props) {
+  return render(
+    <Provider theme={theme}>
+      <Accordion {...props} defaultExpandedKeys={['one']} items={items}>
+        {item => (
+          <Item key={item.key} title={item.title}>
+            {item.children}
+          </Item>
+        )}
+      </Accordion>
+    </Provider>
+  );
+}
+
+describe('Accordion', function () {
+  it('renders properly', function () {
+    renderComponent();
+    let accordionItems = document.querySelectorAll('.spectrum-Accordion-item');
```

Comment:
We generally try to avoid querying by css selectors in tests. I would do something like this:

```suggestion
    let tree = renderComponent();
    let accordionItems = tree.getAllByRole('heading');
```